### PR TITLE
Panama: Update States & Label

### LINKED ISSUE #31160: States for Panamá
**Is your feature request related to a problem? Please describe.**
The out of the box experience of woocommerce for the bigger countries is waaay different from the ones in the development world.
We need on latinamerican countries to have a better out of the box experience. And the only i humbly help is giving by default states for shipping.

As i described on https://github.com/woocommerce/woocommerce/issues/30375

**Describe the solution you'd like**
Add the different shipping states for the country, and change the 'state' label for the one corresponding to this country.
This country uses [Provincia](https://es.wikipedia.org/wiki/Anexo:Provincias_y_comarcas_ind%C3%ADgenas_de_Panam%C3%A1) as 'State'

**Describe alternatives you've considered**
I am chilean, but i can bet that on our sibling countries, they have the same problem. A lot of custom plugins that gives the geographical divisions for the country.

**Additional context**
I've talked to different friends around the continent to get a better idea, but because every country have different ways to set shipping costs around, this is a mild solution.
Like chile, we use Comunas (cities) rather than Regiones (states) for shipping. And how @masteradhoc pointed out, woo cannot have cities for calculating shipping costs. That will be the ultimate solution for everyone.

But at least, this will help latinamerican users to have a better platform experience with Woo, which i love <3.

### PR DESCRIPTION
**Pull request by masteradhoc**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #31160.

### How to test the changes in this Pull Request:

1. add a product on your shop page
2. go to the cart -> click on next to get to the checkout
3. on the checkout choose country
4. check label / states of state field

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry
Change Label of States for Panama & Add States

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:

* [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce/i18n/states.php [hunk 1]
```diff
@@ -1186,6 +1186,21 @@
...
+		'PA-1' => __( 'Bocas del Toro', 'woocommerce' ),
+		'PA-2' => __( 'Coclé', 'woocommerce' ),
+		'PA-3' => __( 'Colón', 'woocommerce' ),
+		'PA-4' => __( 'Chiriquí', 'woocommerce' ),
+		'PA-5' => __( 'Darién', 'woocommerce' ),
+		'PA-6' => __( 'Herrera', 'woocommerce' ),
+		'PA-7' => __( 'Los Santos', 'woocommerce' ),
+		'PA-8' => __( 'Panamá', 'woocommerce' ),
+		'PA-9' => __( 'Veraguas', 'woocommerce' ),
+		'PA-10' => __( 'Panamá Oeste', 'woocommerce' ),
```
Comment (jeffstieler) [inline plugins/woocommerce/i18n/states.php]: I believe this should be the English name, "West Panamá".

**Review by jeffstieler — CHANGES_REQUESTED** [review]
Just need a small tweak here to use the English name in the translation function. Thanks for taking the time to contribute this!

File: plugins/woocommerce/i18n/states.php (hunk 1 above)
Comment (masteradhoc) [reply plugins/woocommerce/i18n/states.php]: Thanks @jeffstieler for all the comments and checks for the PRs. Appreciate the feedback.

**Review by jeffstieler — APPROVED** [review]
:shipit:

**Pull request merged**